src/main/java/tech/gruppone/stalker/server/repositories/UserRepository.java

```diff
@@ -3,13 +3,20 @@
 import org.springframework.data.r2dbc.repository.Modifying;
 import org.springframework.data.r2dbc.repository.Query;
 import org.springframework.data.repository.reactive.ReactiveCrudRepository;
+import reactor.core.publisher.Flux;
 import reactor.core.publisher.Mono;
 import tech.gruppone.stalker.server.model.db.UserDao;
 
 public interface UserRepository extends ReactiveCrudRepository<UserDao, Long> {
 
   public Mono<UserDao> findById(Long id);
 
+  @Query("select * from Users")
+  public Flux<UserDao> findAll();
+
+  @Query("SELECT * FROM User u WHERE u.email = :email")
+  public Mono<UserDao> findByEmail(String email);
+
   @Modifying
   @Query(
       "DELETE Users, UserData FROM Users INNER JOIN UserData WHERE Users.id = UserData.userId AND UserData.userId = :id")
```

Comment:
Cancellare i metodi:

- public Mono<UserDao> findById(Long id);
-  public Flux<UserDao> findAll();
-  public Mono<Void> deleteUserById(Long id);

Tenere **solo** `public Mono<UserDao> findByEmail(String email);`, in quanto è l'unico metodo utilizzato per /user/login (e quindi anche in questa branch)

L'eliminazione dei suddetti metodi è utile per evitare conflitti una volta che si fa il merge nel master